SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -648,6 +649,27 @@ bool USpatialNetDriver::ClientCanSendPlayerSpawnRequests()
 void USpatialNetDriver::OnGSMQuerySuccess()
 {
 	StartupClientDebugString.Empty();
+
```

Comment:
Driveby - could you rename OnGSMQuerySuccess to ClientOnGSMQuerySuccess. Helps clarify who we expect this to be called on.